server/services/userService.ts

```diff
@@ -7,17 +7,17 @@ import { User } from '../@types/CvlUserDetails'
 
 export default class UserService {
   constructor(
-    private readonly hmppsAuthClient: HmppsAuthClient,
+    private readonly manageUsersApiClient: ManageUsersApiClient,
     private readonly prisonApiClient: PrisonApiClient,
     private readonly communityService: CommunityService
   ) {}
 
-  async getAuthUser(user: User): Promise<AuthUserDetails> {
-    return this.hmppsAuthClient.getUser(user)
+  async getUser(user: User): Promise<PrisonUserDetails> {
+    return this.manageUsersApiClient.getUser(user)
   }
 
-  async getAuthUserEmail(user: User): Promise<AuthUserEmail> {
-    return this.hmppsAuthClient.getUserEmail(user)
+  async getUserEmail(user: User): Promise<PrisonUserEmail> {
```

Comment:
renamed following client rename from `hmppsAuthClient` to `manageUsersApiClient`